0.data-download/data_downloader.ipynb

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Jupyter notebook files (`.ipynb`) are [JSON formatted](https://ipython.org/ipython-doc/3/notebook/nbformat.html), which makes them difficult to review.

For this reason, we always include two copies of all notebooks:

1. the `.ipynb` version you've included here
2. an `nbconverted` `.py` version you need to add

To create the `.py` version, run the following command:

```bash
# Activate your conda environment
conda activate __NAME__

# Navigate to the proper folder
cd 0.data-download

# Run nbconvert to create a .py file
jupyter nbconvert --to=script --FilesWriter.build_directory=scripts/nbconverted *.ipynb
```

Please add, commit, push this nbconverted file to this PR, and I will review this file.